src/os/bluestore/BlueStore.cc

```diff
@@ -7756,6 +7785,14 @@ int BlueStore::_open_db_and_around(bool read_only, bool to_repair)
   if (r < 0)
     goto out_fm;
 
+  if (has_null_manager() &&
```

Comment:
I think better place for this block would be inside `_init_alloc()`. There already that null-fm mess.

Reply:
Fixed